modules/unsupported/css/src/test/java/org/geotools/styling/css/ParserSyntheticTest.java

```diff
@@ -820,4 +830,51 @@ public void testMultiNestedSelectors() {
         assertThat(selector, instanceOf(clazz));
         return clazz.cast(selector);
     }
+
+    @Test
+    public void testDynamicColorMapEntryFromCss() throws Exception {
+        String css =
+                "* { raster-channels: 'auto'; raster-color-map: color-map-entry("
+                        + "[env('color','#FF0000')], "
+                        + "[env('min', 1) + 2 * (env('max',10) - env('min',1)) / 3],"
+                        + "[env('customOpacity', 1)])  ;}";
+        LinearColorMap colorMap = buildColorMapFromCss(css);
```

Comment:
So this builds the final result and checks the default values for env are respected?
I would also verify that the ColorMap/ColorMapEntries have the expected structure, similar to the one we'd get out of a SLD parse.

Reply:
Yep. it does 2 passes.
one checking for the default value and one right after having set some ENV values so that the result get different.
I have updated the test to check that the colormap has the expected structure.